.bazelrc

```diff
@@ -49,12 +41,16 @@ test --build_tests_only
 
 # Show all the problems
 test --test_output=errors
+build --verbose_failures
 
 # To use a clang compiler, invoke Bazel with `--config=clang`.
 build:clang --repo_env=CC=clang
 build:clang --repo_env=CXX=clang++
 build:clang --linkopt="-fuse-ld=lld"
 
+# Work around https://github.com/bazelbuild/rules_rust/issues/2125
+build --action_env=CARGO_BAZEL_REPIN=1
```

Comment:
Is this bzlmod related? I don't observe this on the main branch when I test examples (locally).

Reply:
This is indeed required because the lockfile had different shas in Workspace and Bzlmod, see the linked issue